.flake8

```diff
@@ -0,0 +1,14 @@
+[flake8]
+exclude =
+  migrations,
+  manage.py,
+  settings.py
+  .tox,
+  .git,
+  __pycache__,
+  build,
+  dist,
+  *.pyc,
+  *.egg-info,
+  .cache,
+  .eggs
```

Comment:
What's this for?

Reply:
Flake8 is for linting the code, so in this file I'm telling flake8 to ignore this files. I don't know if flake8 is required because on the porting guide you shared us there was a section called linters.